packages/editor-ui/src/constants.ts

```diff
@@ -737,3 +737,12 @@ export const MOUSE_EVENT_BUTTONS = {
 	BROWSER_BACK: 8,
 	BROWSER_FORWARD: 16,
 } as const;
+
+/**
+ * Urls used to route users to the right template repository
+ */
+export const TEMPLATES_URLS = {
+	DEFAULT_API_HOST: 'https://api.n8n.io/api/',
+	BASE_WEBSITE_URL: 'https://n8n.io/workflows',
+	UTM_QUERY: `utm_source=n8n_app&utm_medium=template_library&utm_instance=${window.location.host}`,
```

Comment:
should we use some env variable instead? for example, n8n can be deployed behind a path/subfolder https://github.com/n8n-io/n8n/blob/a822588012b66711f37d3ca363fcd04742eda1b8/docker/compose/subfolderWithSSL/.env#L9

Reply:
Forgot about this, would it be enough to use `window.location.host` + `window.location.pathname`?

Reply:
no because the path name could be /workflows or /demo or something.